aws/rust-runtime/aws-config/examples/imds.rs

```diff
@@ -3,18 +3,23 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-use aws_config::imds::Client;
-use std::error::Error;
-
 /// IMDSv2 client usage example
 ///
 /// The IMDS client is used with `aws-config` to load credentials and regions, however, you can also
 /// use the client directly. This example demonstrates loading the instance-id from IMDS. More
 /// fetures of IMDS can be found [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html)
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn Error>> {
+#[cfg(feature = "imds")]
```

Comment:
you can also make the example require imds in Cargo.toml

Reply:
I didn't know this was possible. Fixed.